docs/pages/web/dropdown/show_case.js

```diff
@@ -0,0 +1,536 @@
+      const text = \`item \${index}\`;
+      return (
+        <Dropdown.Item
+          key={text}
+          onSelect={handleSelect}
+          option={{ value: text, label: text }}
+          selected={selectedElement}
+        />
+      );
+    });
+
+  return (
+    <Box padding={8} height={500} overflow="scrollY" width="100%">
+      <Box height={1000}>
+        <ScrollBoundaryContainer>
+          <Button
+            text="Menu open button"
+            onClick={() => setIsOpen(!isOpen)}
+            ref={anchorRef}
+            selected={isOpen}
+            size="lg"
+          />
+          {isOpen && (
+            <Dropdown
+              isWithinFixedContainer
+              anchor={anchorRef.current}
+              id="demo-dropdown-example"
+              onDismiss={() => setIsOpen(!isOpen)}
+              idealDirection="up"
+            >
+              {preRenderItems()}
+            </Dropdown>
+          )}
+        </ScrollBoundaryContainer>
+      </Box>
+    </Box>
+  );
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Scrollable container 2"
+            description="Dropdown inner scrollable container, should have a 'ScrollBoundaryContainer' around the dropdown implementation"
+            defaultCode={`
+function ScrollableContainerB() {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [selectedElement, setSelectedElement] = React.useState(null);
+  const [elements, setElements] = React.useState([]);
+  const anchorRef = React.useRef(null);
+
+  React.useEffect(() => {
+    setElements(new Array(20).fill(undefined));
+  }, [setElements]);
+
+  const handleSelect = ({ item }) => {
+    setSelectedElement(item);
+  };
+
+  const preRenderItems = () =>
+    elements.map((item, index) => {
+      const text = \`item \${index}\`;
+      return (
+        <Dropdown.Item
+          key={text}
+          onSelect={handleSelect}
+          option={{ value: text, label: text }}
+          selected={selectedElement}
+        />
+      );
+    });
+
+  return (
+    <Box padding={8} height={500} overflow="scrollY" color="successBase" width="100%">
+      <Box height={300} color="errorBase" overflow="hidden">
+        <ScrollBoundaryContainer>
+          <Button
+            text="Menu open button"
+            onClick={() => setIsOpen(!isOpen)}
+            ref={anchorRef}
+            selected={isOpen}
+            size="lg"
+          />
+          {isOpen && (
+            <Dropdown
+              anchor={anchorRef.current}
+              id="demo-dropdown-example"
+              onDismiss={() => setIsOpen(!isOpen)}
+              idealDirection="up"
+            >
+              {preRenderItems()}
+            </Dropdown>
+          )}
+        </ScrollBoundaryContainer>
+      </Box>
+    </Box>
+  );
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Page Header"
```

Comment:
https://www.loom.com/share/e40c199ed6eb465aa843732c42843faa

Reply:
Something is wrong to me when I access the PageHeader on our current online documentation, as you can see bellow, so I can't fix this description, to keep de focus on BUG's I removed this example too.
![PageHeaderNotWorking](https://user-images.githubusercontent.com/16820917/202543967-42cb833b-cebc-42ca-98f8-ba4390c27e7d.gif)